JOSESwift/Sources/Encrypter.swift

```diff
@@ -76,11 +76,11 @@ public struct Encrypter<KeyType> {
     ///   - contentEncyptionAlgorithm: The algorithm used to encrypt the JWE's payload.
     /// - Returns: A fully initialized `Encrypter` or `nil` if provided key is of the wrong type.
     public init?(keyEncryptionAlgorithm: AsymmetricKeyAlgorithm, keyEncryptionKey kek: KeyType, contentEncyptionAlgorithm: SymmetricKeyAlgorithm) {
+        guard type(of: kek) is RSAEncrypter.KeyType.Type else {
+            return nil
+        }
         switch (keyEncryptionAlgorithm, contentEncyptionAlgorithm) {
-        case (.RSA1_5, .A256CBCHS512) :
-            guard type(of: kek) is RSAEncrypter.KeyType.Type else {
-                return nil
-            }
+        case (.RSA1_5, .A256CBCHS512), (.RSAES_OAEP, .A256CBCHS512) :
```

Comment:
Whitespace before `:`

Reply:
Sorry man, the whitespace before `:` was already there in the `Encrypter.swift` file, line 80 and somehow I assumed that was the convention. Im happy to remove it :)

Reply:
Fixed